src/text_render.rs

```diff
@@ -385,3 +553,29 @@ fn create_oversized_buffer(
 fn zero_depth(_: usize) -> f32 {
     0f32
 }
+
+#[cfg(feature = "custom-glyphs")]
+#[derive(Debug, Clone, Copy, PartialEq)]
+/// The input data to render a custom glyph
+pub struct CustomGlyphInput {
+    /// The unique identifier of the glyph.
+    pub id: crate::CustomGlyphID,
+    /// The size of the glyph in points (not scaled by the text area's scaling factor)
+    pub size: f32,
+    /// The scaling factor applied to the text area.
+    pub scale: f32,
+    /// Binning of fractional X offset
+    pub x_bin: cosmic_text::SubpixelBin,
```

Comment:
This won't work because you would need to have the position as part of the cache key, which would potentially lead to a whole lot of atlas entries being created.

I think the better solution is to add a flag to `CustomGlyph` saying whether or not you want to snap the glyph to the nearest whole pixel.